docs/source/configuration.rst

```diff
@@ -212,30 +209,29 @@ be a :code:`.sql` itself.
     be a requirement that the *templating* is accurate - only so far as that
     is required to enable the *parsing* and *linting* to be helpful.
 
-Builtin Macro Blocks
-^^^^^^^^^^^^^^^^^^^^
+Dbt Project Configuration
+-------------------------
 
-One of the main use cases which inspired *sqlfluff* as a a project was `dbt`_.
-It uses jinja templating extensively and leads to some users maintaining large
-repositories of sql files which could potentially benefit from some linting.
+dbt is not the default templater for *sqlfluff* (it is Jinja). In order
+to get started using *sqlfluff* with a Dbt project you will need the following
+configuration:
 
-*Sqlfluff* anticipates this use case and provides some built in macro blocks
-in the `Default Configuration`_ which assist in getting started with `dbt`_
-projects. In particular it provides mock objects for:
+In *.sqlfluff*:
 
-* *ref*: The mock version of this provided simply returns the model reference
-  as the name of the table. In most cases this is sufficient.
-* *config*: A regularly used macro in `dbt`_ to set configuration values. For
-  linting purposes, this makes no difference and so the provided macro simply
-  returns nothing.
+.. code-block:: cfg
 
-.. note::
+    [sqlfluff]
+    templater = dbt
+    # DBT templating does not keep trailing new lines (L009)
+    exclude_rules = L009
```

Comment:
Must we exclude L009?  I didn't in my local config and was able to successfully lint

Reply:
FWIW I did get lint issues with `L009` when linting my project

Reply:
which version of dbt were you using @sethwoodworth ?

Reply:
dbt 0.18.1